megamek/src/megamek/common/Mech.java

```diff
@@ -5357,7 +5367,10 @@ public double getCost(boolean ignoreAmmo) {
             cockpitCost = 500000;
         } else if (getCockpitType() == Mech.COCKPIT_SMALL_COMMAND_CONSOLE) {
             // The cost is the sum of both small and command console
-            cockpitCost = 675000;            
+            cockpitCost = 675000; 
+        } else if (getCockpitType() == Mech.COCKPIT_TORSO_MOUNTED_INDUSTRIAL) {
+            cockpitCost = 750000;
+            // There is no cost for the industrial variant, and the cost is same as torso-mounted cockpit
```

Comment:
If the comment is true this should be rolled into the logic for COCKPIT_TORSO_MOUNTED.